### PR TITLE
CASSANDRA-16039 - FQL replay has option to ignore DDL statements

### PR DESCRIPTION
**Pull request by smiklosovic**

### CODE REVIEW

File: test/distributed/org/apache/cassandra/distributed/test/FqlReplayDDLExclusionTest.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            final ToolRunner.Runners runners = new ToolRunner.Runners();
+
+            // without --replay-ddl-statements, the replay will fail on insert because underlying table is not there
+            final ToolRunner negativeRunner = runners.invokeClassAsTool("org.apache.cassandra.fqltool.FullQueryLogTool",
+                                                                        "replay",
+                                                                        "--keyspace", "fql_ks",
+                                                                        "--target", "127.0.0.1",
+                                                                        "--", temporaryFolder.getRoot().getAbsolutePath());
+
+            assertEquals(0, negativeRunner.getExitCode());
```
Comment (dcapwell) [inline test/distributed/org/apache/cassandra/distributed/test/FqlReplayDDLExclusionTest.java]: if all queries fail we return 0?  Feels unexpected...

File: test/distributed/org/apache/cassandra/distributed/test/FqlReplayDDLExclusionTest.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            final ToolRunner positiveRunner = runners.invokeClassAsTool("org.apache.cassandra.fqltool.FullQueryLogTool",
+                                                                        "replay",
+                                                                        "--keyspace", "fql_ks",
+                                                                        "--target", "127.0.0.1",
+                                                                        // important
+                                                                        "--replay-ddl-statements",
+                                                                        "--", temporaryFolder.getRoot().getAbsolutePath());
+
+            assertEquals(0, positiveRunner.getExitCode());
+            assertTrue(node.executeInternalWithResult("SELECT * from fql_ks.fql_table;").hasNext());
```
Comment (dcapwell) [inline test/distributed/org/apache/cassandra/distributed/test/FqlReplayDDLExclusionTest.java]: can you also assert the rows?

Comment (dcapwell) [reply test/distributed/org/apache/cassandra/distributed/test/FqlReplayDDLExclusionTest.java]: row utils will handle it for you, so just need to show that only a single row with the value of 1 is present.

```
assertRows(node.executeWithResult("SELECT * from fql_ks.fql_table"),
                       QueryResults.builder().row(1).build());
```

that will test that only one row exists and the only value is `1`

File: test/distributed/org/apache/cassandra/distributed/test/FqlReplayDDLExclusionTest.java (hunk 1 above)
Comment (smiklosovic) [reply test/distributed/org/apache/cassandra/distributed/test/FqlReplayDDLExclusionTest.java]: yes, it does.

**Comment by smiklosovic** [comment]
Closing as this was merged here https://github.com/apache/cassandra/commit/00c22232d96b6e592fb1d1f32a6b649b49eceb34

**Pull request closed without merging**